blocks/header-nav-chain-block/chains/header-nav-chain-block/_children/horizontal-links/default.jsx

```diff
@@ -39,6 +38,23 @@ const HorizontalLinksBar = ({ hierarchy, navBarColor, showHorizontalSeperatorDot
 	// the interior links and spans need to contrast with nav color scheme
 	const readableContrastingColor = navBarColor === "light" ? "#000" : "#fff";
 
+	const LinkBuilder = ({ item }) => {
+		const linkUrl = item.node_type === "link" ? item.url : item._id;
+		const linkText = item.node_type === "link" ? item.display_name : item.name;
+		const externalUrl = /(http(s?)):\/\//i.test(linkUrl);
```

Comment:
This only tests if the link has http(s) to see if it's external.  It should include a domain validation.  I think I've seen this elsewhere.  Was it copied from somewhere?

Reply:
Thinking this should be a utility, that's mainly my concern.

Reply:
The `Link` component has this logic built into it